tests/contrib/minikube_tests/integration/test_kubernetes_executor_integration.py

```diff
@@ -1,67 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import time
-import unittest
-from uuid import uuid4
-
-from tests.contrib.minikube_tests.integration.airflow_controller\
-    import DagRunState, RunCommandError, \
-    dag_final_state, get_dag_run_state, kill_scheduler, run_command, run_dag
-
-try:
-    run_command("kubectl get pods")
-except RunCommandError:
-    SKIP_KUBE = True
-else:
-    SKIP_KUBE = False
-
-
-class KubernetesExecutorTest(unittest.TestCase):
-    @unittest.skipIf(SKIP_KUBE,
-                     'Kubernetes integration tests are unsupported by this configuration')
-    def test_kubernetes_executor_dag_runs_successfully(self):
-        dag_id, run_id = "example_python_operator", uuid4().hex
-        run_dag(dag_id, run_id)
-        state = dag_final_state(dag_id, run_id, timeout=120)
-        self.assertEquals(state, DagRunState.SUCCESS)
-
-    @unittest.skipIf(SKIP_KUBE,
-                     'Kubernetes integration tests are unsupported by this configuration')
-    def test_start_dag_then_kill_scheduler_then_ensure_dag_succeeds(self):
```

Comment:
I think that testing the Scheduler killing + restarting is a pretty important feature for the executor. Could you please re-implement this test in the integration tests?